src/renderer/features/fellowship-overview/components/MembersTab/MembersEmptyState.tsx

```diff
@@ -0,0 +1,53 @@
+import { useI18n } from '@/shared/i18n';
+import { Button, FootnoteText, Icon, SmallTitleText } from '@/shared/ui';
+import { Surface } from '@/shared/ui-kit';
+
+type Props = {
+  searchQuery: string;
+  onClearSearch: () => void;
+  hasActiveFilters?: boolean;
+};
+
+export const MembersEmptyState = ({ searchQuery, onClearSearch, hasActiveFilters }: Props) => {
+  const { t } = useI18n();
+
+  const isSearchOnly = searchQuery && !hasActiveFilters;
+  const isFilterOnly = hasActiveFilters && !searchQuery;
+
+  const getTitle = () => {
+    return t('fellowship.activityFeed.activityModal.no-results.title');
+  };
+
+  const getDescription = () => {
+    if (isSearchOnly) {
+      return t('fellowship.activityFeed.activityModal.no-results.search', {
+        query: searchQuery,
+      });
+    }
+    if (isFilterOnly) {
+      return t('fellowship.activityFeed.activityModal.no-results.filters');
+    }
+    return t('fellowship.activityFeed.activityModal.no-results.general');
+  };
```

Comment:
Maybe it's better to make constants instead of functions that are called on every render?
```suggestion
const title = t('fellowship.activityFeed.activityModal.no-results.title');

const description = useMemo(() => {
  if (isSearchOnly) {
    return t('fellowship.activityFeed.activityModal.no-results.search', {
      query: searchQuery,
    });
  }
  if (isFilterOnly) {
    return t('fellowship.activityFeed.activityModal.no-results.filters');
  }
  return t('fellowship.activityFeed.activityModal.no-results.general');
}, [isSearchOnly, isFilterOnly, searchQuery, t]);
```

Reply:
Fixed